### ISSUE TITLE
Amazon incorrectly reporting out of stock

### ISSUE THREAD

**Issue by HammerProgrammer** [jef/streetmerchant#457]
### Description

If you leave the script running for a long and then stop and run a test with the test model it will report out of stock
but if you change your ip and run the test again it works.

### Possible solution

It either needs to report the real state of the product or throw an error when it stops being able to do so.

**Comment by jef** [comment]
This has to do with the way we are logging, so it's not necessarily a bug. I think we should open this up for discussion though to figure out what want to print for these cases.

Thanks!

**Comment by admon84** [comment]
I tried to recreate this behavior and the first thing I noticed the the test model card is out of stock currently.  

I updated the test model to an in-stock card in a local branch to keep testing and couldn't recreate the behavior.  I'm reliably getting in-stock versus out-of-stock as expected after changing the test model card.  I ran it for 30 minutes on amazon-only on 30 series cards (all out of stock) and then switched to running the test model card for 10 minutes and it always showed "in stock" for me.
